### PR TITLE
Remove Texture and CocoaPods

### PR DESCRIPTION
**Pull request by denkeni**
This closes https://github.com/Ptt-official-app/Ptt-iOS/issues/59.

### CODE REVIEW

File: Ptt/Flows/LoginFlow/Controllers/LoginSubView.swift [hunk 1]
```diff
@@ -191,33 +134,29 @@ extension LoginViewController {
...
-            if let tf = self.tfPassword.view as? LoginTextField {
-                tf.warning(msg: L10n.notFinish)
-            }
+            self.tfPassword.warning(msg: L10n.notFinish)
             return
         }
 
         self.btnLogin.isEnabled = false
         APIClient.shared.login(account: account, password: passwd) { result in
             self.btnLogin.isEnabled = true
```
Comment (xxi511) [inline Ptt/Flows/LoginFlow/Controllers/LoginSubView.swift:158]: `-[UIButton setEnabled:] must be used from main thread only`

File: Ptt/API/APIClient.swift [hunk 2]
```diff
@@ -201,9 +201,11 @@ extension APIClient: APIClientProtocol {
...
             // Percent encoding is automatically done with RFC 3986
-            urlComponent.queryItems = [
-                URLQueryItem(name: "start_idx", value: startIdx)
-            ]
+            if let startIdx {
+                urlComponent.queryItems = [
+                    URLQueryItem(name: "start_idx", value: startIdx)
+                ]
+            }
         case .go_bbs(boardID: let boardID):
```
Comment (xxi511) [inline Ptt/API/APIClient.swift:209]: out of curios, do we have backend approach decision?

Comment (denkeni) [reply Ptt/API/APIClient.swift:209]: Not yet. We’re currently focusing on implementing with go_pttbbs, but still open to other solutions.

File: Ptt/Flows/LoginFlow/Controllers/LoginSubView.swift (hunk 1 above)
Comment (denkeni) [reply Ptt/Flows/LoginFlow/Controllers/LoginSubView.swift]: Thanks! I've fixed the issue.

**Comment by xxi511** [comment]
FYI, the button is blocked by keyboard    


Simulator Screenshot - iPhone 11 - 2023-04-29 at 22 52 10

**Comment by denkeni** [comment]
> FYI, the button is blocked by keyboard nvm, user can dismiss keyboard

Sure. The related function [scrollViewToFitKeyboard](https://github.com/Ptt-official-app/Ptt-iOS/blob/20d0971c1aa72a425a42ea2674db2a6c671d5763/Ptt/Flows/LoginFlow/Controllers/LoginViewController.swift#L487L500) was clearly incomplete, so there’s definitely room for improvement.

**Pull request merged**